lib/validations/url_create.ts

```diff
@@ -1,28 +1,36 @@
-import { NextRequest } from "next/server";
 import { urlSchema } from "../zod/url";
+import cheerio from 'cheerio';
 
-import { IUrlCreateReq } from "@/interfaces/url";
+const validateURLCreateReq = async (formdata: FormData) => {
+  try {
+    const long_url = formdata.get("longUrl");
+    let title:any = formdata.get("title");
 
-const validateURLCreateReq =  (formdata: FormData) => {
-    try {
-      
-      const long_url = formdata.get("longUrl");
-      const title = formdata.get("title");
-      
-      // validation of title need to be done
-      const errors = urlSchema.safeParse({
-        long_url,
-      });
-      return { title,long_url, status: errors.success, msg: errors.error };
-    } catch (e) {
-      return {
-        title: "",
-        long_url: "",
-        status: false,
-        msg : JSON.stringify(e)
-      };
+    // validation of title need to be done
+    const errors = urlSchema.safeParse({
+      long_url,
+    });
+
+    const urlHit: any = long_url;
```

Comment:
Why are you creating a new variable use long_url itself?

Reply:
because it was causing typescript error in fetch request. As fetch request accepts URL | String as params and type of long_url is FormDataValue | undefined